IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -2281,6 +2287,29 @@ private void OpenTiffToImageSharp(ReadOnlySpan<byte> bytes)
             }
         }
 
+        private Compression SetTiffCompression(Tiff tif)
+        {
+            Compression tifCompression = tif.GetField(TiffTag.COMPRESSION) != null && tif.GetField(TiffTag.COMPRESSION).Length > 0
```

Comment:
nit: keep variables referring to `tiff` the same in regards to how you spell it (e.g. this ought to be `tiffCompression`)